gematria/granite/python/graph_builder_model_base.py

```diff
@@ -143,6 +160,21 @@ def __init__(
 
     self._special_tokens_tensor = None
 
+    self._instruction_node_mask = tf.placeholder(
+        dtype=tf.dtypes.bool,
+        shape=(None,),
+        name=GraphBuilderModelBase.INSTRUCTION_NODE_MASK_TENSOR_NAME,
+    )
```

Comment:
The two `tf.placeholder()`s should be created in `self._create_tf_graph()`, along with the rest of the TF ops.

I remember that in the past creating tensors in the constructor led to different issues with having TF ops in two different TF graphs (which then caused execution issues).